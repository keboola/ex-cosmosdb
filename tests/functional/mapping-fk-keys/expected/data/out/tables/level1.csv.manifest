{
    "columns": [
        "a",
        "b",
        "level2-a",
        "level2-b",
        "output-csv_pk"
    ],
    "primary_key": [],
    "incremental": false
}
