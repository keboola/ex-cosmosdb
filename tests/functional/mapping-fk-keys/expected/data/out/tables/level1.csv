"123","123","be5e25e72e6b2a5454dc99f9cc830c21","be5e25e72e6b2a5454dc99f9cc830c21","4a59fa3c94f060c5060c16a37d8a08b1"
"123","123","8548ce314658dfbc022ca94635cf1905","8548ce314658dfbc022ca94635cf1905","d725966bce29c19dae4131145f26e529"
"456","123","177bc26b5ae19099c845ed74d094eed6","177bc26b5ae19099c845ed74d094eed6","87a69f76b73ba3008c4e800e9c66d9f1"
"456","123","cc281168d52b3c39ad594a56c94548af","cc281168d52b3c39ad594a56c94548af","a16939320e0ba853a22833ca84fca80d"
